.gitattributes

```diff
@@ -1,3 +1,4 @@
 benchcases/** linguist-detectable=false
 fixtures/** linguist-detectable=false
 webpack-test/** linguist-detectable=false
+* text=auto
```

Comment:
I'm not familiar with this. Does it achieve the goal of enforcing `line-style: LF`?

Reply:
The main purpose is to not change every lines of every files.

It will make the codes in remote repo(.i.e, git) ending with LF, and any new clone will be LF, because the files in the remote is LF.

But for your local files, this line does not work. It just tells git to ignore the ending of line as diff. 

You could run `git add --renormalize .`(as suggested in the article link), and install extension "EditorConfig.EditorConfig" to make VSCode to respect ".editorconfig" file.

You could reference this great article https://www.aleksandrhovhannisyan.com/blog/crlf-vs-lf-normalizing-line-endings-in-git/